.github/workflows/build.yml

```diff
@@ -0,0 +1,195 @@
+      run: |
+        #!/bin/bash
+        set -e
+        for f in linux_*.sh; do
+          if [[ "$f" == *_cross.sh ]] || [[ "$f" == *_regular.sh ]]
+          then
+            continue
+          fi
+          bash "$f"
+        done
+      working-directory: cb/bld
+    - name: Build (Android)
+      if: startsWith(matrix.os, 'ubuntu')
+      run: |
+        set -e
+        cd android_e_sqlite3
+        ndk-build
+        mkdir -p ../bin/e_sqlite3/android
+        cp -r libs/* ../bin/e_sqlite3/android
+        cd ../android_e_sqlcipher
+        ndk-build
+        mkdir -p ../bin/e_sqlcipher/android
+        cp -r libs/* ../bin/e_sqlcipher/android
+      working-directory: cb/bld
+    - name: Build (macOS, iOS, tvOS)
+      if: startsWith(matrix.os, 'macos')
+      run: |
+        set -e
+        for f in mac_*.sh ios_*.sh tvos_*.sh; do
+          bash "$f"
+        done
+      working-directory: cb/bld
+    - name: Build (Windows)
+      if: startsWith(matrix.os, 'windows')
+      run: |
+        $files = Get-ChildItem *.bat -Exclude "win_e_sqlite3.bat","win_e_sqlcipher.bat"
+        foreach ($f in $files) {
+          cmd /C $f
+        }
+      working-directory: cb/bld
+    - uses: actions/upload-artifact@v2
+      with:
+        name: bin-${{ matrix.os }}
+        path: cb/bld/bin
+
+  test:
+    needs: build
+    if: ${{ github.event_name == 'pull_request' }}
+    strategy:
+      matrix:
+        os: [windows-latest, macos-latest]
+    runs-on: ${{ matrix.os }}
+    steps:
+    - uses: actions/checkout@v2
+      with:
+        path: "cb"
+    - uses: actions/checkout@v2
+      with:
+        fetch-depth: 0
+        repository: ericsink/SQLitePCL.raw
+        path: "SQLitePCL.raw"
+    - name: Clean output directory
+      if: startsWith(matrix.os, 'ubuntu') || startsWith(matrix.os, 'macos')
+      run: rm -fr bin
+      working-directory: cb/bld
+    - name: Clean output directory
+      if: startsWith(matrix.os, 'windows')
+      run: rm bin -r -fo
+      working-directory: cb/bld
+    - uses: actions/download-artifact@v2
+      with:
+        name: bin-ubuntu-latest
+        path: cb/bld/bin
+    - uses: actions/download-artifact@v2
+      with:
+        name: bin-macos-latest
+        path: cb/bld/bin
+    - uses: actions/download-artifact@v2
+      with:
+        name: bin-windows-latest
+        path: cb/bld/bin
+    - name: Setup .NET 5
+      uses: actions/setup-dotnet@v1
+      with:
+        dotnet-version: 5.0.x
+    - name: Setup .NET 3.1
+      uses: actions/setup-dotnet@v1
+      with:
+        dotnet-version: 3.1.x
+    - name: Setup .NET 2.1
+      uses: actions/setup-dotnet@v1
+      with:
+        dotnet-version: 2.1.x
+    - name: Add msbuild to PATH
+      if: startsWith(matrix.os, 'windows')
+      uses: microsoft/setup-msbuild@v1.0.2
+    - name: Install T4
+      run: dotnet tool install --global dotnet-t4
+    - name: Build
+      run: |
+        cd SQLitePCL.raw/build
+        dotnet run
```

Comment:
This is pretty much verbatim copy of the GitHub Actions pipeline on SQLitePCL.Raw. The only difference is that the `cb` binaries are replaced with build artifacts from previous step.